examples/angular/basic-external-state/src/styles.css

```
@@ -0,0 +1,60 @@
+  height: 1rem;
+}
+
+.controls,
+.inline-controls {
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+}
+
+.demo-button {
+  border: 1px solid currentColor;
```

Stylelint (17.11.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)